backend/user-service/controllers/user-controller.ts

```diff
@@ -106,9 +106,16 @@ async function loginUser(req: Request, res: Response) {
   }
 }
 
-async function getUserByUserId(req: Request, res: Response) {
-  const { uid } = req.body;
+async function getUserInfo(req: Request, res: Response) {
+  const queryUidString = req.query.uid;
+  console.log(queryUidString);
```

Comment:
I understand that printing does help in debugging, but I think once the issue is resolved, you need to remove the console.log message to avoid leaking information or wasting resources.